src/Conversions/include/BipedalLocomotion/Conversions/ManifConversions.h

```diff
@@ -0,0 +1,76 @@
+ * distributed under the terms of the GNU Lesser General Public License v2.1 or any later version.
+ */
+
+#ifndef BIPEDAL_LOCOMOTION_MANIF_CONVERSIONS_H
+#define BIPEDAL_LOCOMOTION_MANIF_CONVERSIONS_H
+
+#include <iDynTree/Core/Transform.h>
+#include <iDynTree/Core/EigenHelpers.h>
+#include <manif/manif.h>
+
+
+namespace BipedalLocomotion
+{
+namespace Conversions
+{
+    /**
+     * @brief Convert rotation matrix and translation vector to manif SE3 object
+     *
+     * @param rotation reference to 3x3 Eigen matrix
+     * @param translation reference of 3x1 Eigen matrix
+     * @param pose reference to manif SE3 object
+     */
+    template <class Scalar>
+    void eigenRotTrans2ManifPose(const Eigen::Matrix<Scalar, 3, 3>& rotation,
+                                 const Eigen::Matrix<Scalar, 3, 1>& translation,
+                                 manif::SE3<Scalar>& pose)
+    {
+        Eigen::Quaternion<Scalar> quat = Eigen::Quaternion<Scalar>(rotation);
+        quat.normalize(); // SO3 constructor expects normalized quaternion
+        pose = manif::SE3<Scalar>(translation, quat);
+    }
+
+    template void eigenRotTrans2ManifPose<double>(const Eigen::Matrix<double, 3, 3>& rotation,
+                                                  const Eigen::Matrix<double, 3, 1>& translation,
+                                                  manif::SE3<double>& pose);
+    template void eigenRotTrans2ManifPose<float>(const Eigen::Matrix<float, 3, 3>& rotation,
+                                                  const Eigen::Matrix<float, 3, 1>& translation,
+                                                  manif::SE3<float>& pose);
+
+    /**
+     * @brief Convert rotation matrix and translation vector to manif SE3d object
+     *
+     * @param rotation Eigen ref of 3x3 rotation matrix
+     * @param translation Eigen ref of 3x1 translation vector
+     * @param pose reference to manif SE3d object
+     */
+    inline void eigenRotTrans2ManifPose(Eigen::Ref<const Eigen::Matrix3d> rotation,
+                                        Eigen::Ref<const Eigen::Vector3d> translation,
+                                        manif::SE3d& pose)
+    {
+        Eigen::Quaterniond quat = Eigen::Quaterniond(rotation);
+        quat.normalize(); // SO3 constructor expects normalized quaternion
+        pose = manif::SE3d(translation, quat);
+    }
+
+    /**
+     * @brief convert iDynTree transform object to manif SE3d object
+     *
+     * @param H reference to iDynTree Tranform object
+     * @param pose reference to manif SE3 object
+     */
+    inline void iDynTransform2ManifPose(const iDynTree::Transform& H, manif::SE3d& pose)
+    {
+        BipedalLocomotion::Conversions::eigenRotTrans2ManifPose(iDynTree::toEigen(H.getRotation()),
+                                                                iDynTree::toEigen(H.getPosition()),
+                                                                pose);
+    }
```

Comment:
Also here, we may have also:
```cpp
inline manif::SE3d iDynTransform2ManifPose(const iDynTree::Transform& H)
```

Reply:
Addressed in https://github.com/dic-iit/bipedal-locomotion-framework/pull/138/commits/12d97ab9746193b054a09fd3be5e99ddd0b80836